packages/console/src/containers/AppContent/components/Topbar/index.tsx

```diff
@@ -18,7 +21,8 @@ type Props = {
 
 function Topbar({ className }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const LogtoLogo = isCloud ? CloudLogo : Logo;
+  const theme = useTheme();
+  const LogtoLogo = isCloud ? (theme === Theme.Light ? CloudLogo : CloudLogoDark) : Logo;
```

Comment:
What if it's not cloud env? It seems it will fallback to `Logo`, which is not checking theme, either